Gir_Gtk.toml

```diff
@@ -357,3 +357,10 @@ status = "generate"
 [[object]]
 name = "Gtk.PaperSize"
 status = "generate"
+
+[[object]]
+name = "Gtk.WidgetPath"
+status = "generate"
+ignored_functions = [
+    '''iter_has_q\w+''',
```

Comment:
So this quote from [here](https://github.com/toml-lang/toml#string) is not correct?

> Literal strings are surrounded by single quotes. Like basic strings, they must appear on a single line:

>```
# What you see is what you get.
winpath  = 'C:\Users\nodejs\templates'
winpath2 = '\\ServerX\admin$\system32\'
quoted   = 'Tom "Dubs" Preston-Werner'
regex    = '<\i\c*\s*>'
```